examples/rcbilling-demo/src/pages/success/index.tsx

```diff
@@ -3,19 +3,12 @@ import AppStoreButton from "../../components/AppStoreButton";
 import PlayStoreButton from "../../components/PlayStoreButton";
 import AppLogo from "../../components/AppLogo";
 import LogoutButton from "../../components/LogoutButton";
-import { Purchases } from "@revenuecat/purchases-js";
-
-const isAnonymousUser = () => {
-  const anonymousIDRegex = /^\$RCAnonymousID:([a-f0-9]{32})$/;
-  return anonymousIDRegex.test(Purchases.getSharedInstance().getAppUserId());
-};
-
-const generateDeepLink = () => {
-  const appUserId = Purchases.getSharedInstance().getAppUserId();
-  return `revenuecatbilling://redeem_rcb_purchase?redemption_token=${encodeURIComponent(appUserId)}`;
-};
 
 const SuccessPage: React.FC = () => {
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const redeemUrlParam = urlParams.get("redeem_url");
```

Comment:
there might be a better way, but since this is only for the demo app, I think it's ok to do it this way 👍